README.md

```diff
@@ -1,23 +1,26 @@
 # Ceph - a scalable distributed storage system
 
-Please see https://ceph.com/ for current info.
+See https://ceph.com/ for current information about Ceph.
 
 
 ## Contributing Code
 
-Most of Ceph is dual licensed under the LGPL version 2.1 or 3.0.  Some
-miscellaneous code is under a BSD-style license or is public domain.
-The documentation is licensed under Creative Commons
-Attribution Share Alike 3.0 (CC-BY-SA-3.0).  There are a handful of headers
-included here that are licensed under the GPL.  Please see the file
-COPYING for a full inventory of licenses by file.
+Most of Ceph is dual licensed under the LGPL version 2.1 or 3.0. Some
+miscellaneous code is either public domain or licensed under a BSD-style
+license.
 
-Code contributions must include a valid "Signed-off-by" acknowledging
-the license for the modified or contributed file.  Please see the file
-SubmittingPatches.rst for details on what that means and on how to
+The Ceph documentation is licensed under Creative Commons Attribution Share
+Alike 3.0 (CC-BY-SA-3.0). 
+
+Some headers included in the `ceph/ceph` repository are licensed under the GPL.
+See the file `COPYING` for a full inventory of licenses by file.
+
+All code contributions must include a valid "Signed-off-by" line that
+acknowledges the license for the modified or contributed file. See the
```

Comment:
I don't think I've ever seen a signed-off-by that mentions a license.

Reply:
@anthonyeleven This is precisely what I had hoped you or someone else would say, ratifying my gut feeling and giving me the license I need to cut this out, secure in my judgment through the unbiased, undirected confirmation of another.

Reply:
Now, if we *should* be mentioning a license explicitly, then we need to add that across the board.  I've never noticed a Ceph Signed-off-by with an explicit license. So I would be inclined to remove this, or re-word it to be *implicitly* approving whatever license model Ceph uses.  The latter is what I was originally told that signed-off-by signifies here.